be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -139,8 +139,8 @@ final case class DbActor(
   }
 
   @throws[DbActorNAckException]
-  private def createElectionData(electionId: Hash, keyPair: KeyPair): Unit = {
-    val channel = Channel(s"${ROOT_CHANNEL_PREFIX}private/${electionId.toString}")
+  private def createElectionData(electionId: Hash, keyPair: KeyPair, laoId: Hash): Unit = {
+    val channel = Channel(s"${ROOT_CHANNEL_PREFIX}${laoId.toString}/private/${electionId.toString}")
```

Comment:
do you still need the `/private` segment in the path ? 
Conversely, how are you going to ensure that this key is always interpreted as an election in the database ? 

This question applied before your changes too, I'm just raising a question that could've been addressed by students in past iterations 😄

Reply:
I suppose using `/election` would be better ? At least it would be clearer and more coherent with the other keywords used so far (rollcall, social, reaction ...etc)

Reply:
The purpose of this object `ElectionData` is simply to keep the private key stored somewhere where we cannot access it. 
There is already the channel `root/laoId/electionId` which is defined for the election channel. The problem of storing the `ElectionData` inside this channel, was when we wanted to retrieve messages from the election channel, which is done in the validators or when computing the results of election, anyone could access to the private key. 
To dodge this problem, we decided to create another channel defined with a `private` just to differentiate it from the main election channel. This channel is simply used to store the `ElectionData` where we can find the private key used for the election in question.